Configurations/unix-Makefile.tmpl

```diff
@@ -493,6 +493,14 @@ list-tests:
 
 install: install_sw install_ssldirs install_docs
 
+install_fips: install_sw
+	@[ -n "$(INSTALLTOP)" ] || (echo INSTALLTOP should not be empty; exit 1)
```

Comment:
Isn't this test redundant because of ```install_sw``` ?  Shouldn't this be inline when the library is installed, or move that install here?  And what about ```uninstall_fips``` which should be part of ```uninstall_sw``` ?

Reply:
Good point, thanks!

Reply:
Fixed, plus a slew of other stuff I discovered